src/btree/bt_debug.c

```diff
@@ -1008,22 +1008,17 @@ __debug_update(WT_DBG *ds, WT_UPDATE *upd, bool hexbyte)
 		WT_RET(ds->f(ds, "\t" "txn id %" PRIu64, upd->txnid));
 
 #ifdef HAVE_TIMESTAMPS
-		if (!__wt_timestamp_iszero(upd->timestamp)) {
+		if (!__wt_timestamp_iszero(WT_GET_TIMESTAMP_PTR(upd))) {
```

Comment:
FWIW, I suggest we get rid of `WT_GET_TIMESTAMP_PTR` (it's only used in 7 places), and just spell out the `->timestamp` part.

I'd also prefer a different name for `WT_TIMESTAMP_PTR` (basically, we're conflating a bunch of stuff: adding `->timestamp`, adding `&`, and setting the argument to NULL, and that's hard to think about.

I'd suggest spelling out `->timestamp` and `&`, and then using something like `WT_TIMESTAMP_NULL` for setting the argument to NULL.